### ISSUE TITLE
In game guide not loading

### ISSUE THREAD

**Issue by arilando** [ValveSoftware/Dota2-Gameplay#2549]
### Description

Some of the in game guides i'm using aren't loading. Specifically on some heroes, the guide i have previously selected will not load, while the other guides load just fine. For some heroes the guide i have previously selected also loads just fine. Reinstalling, verifying integrity of game files or clearing steam download cache does not fix the problem. This problem occurs on my desktop, on my laptop the problem does not occur. What can i do about this?

### Screenshots

![Dota 2 bug](https://user-images.githubusercontent.com/25887181/187713853-2c9529f3-dc05-4abc-9422-75429237fe9c.png)

**Comment by basocjk** [comment]
what fixed mine is running steam as administrator.

**Comment by jbmlaird** [comment]
Running as administrator worked for the first game, then had the same issue subsequent games

**Comment by method0G** [comment]
> Running as administrator worked for the first game, then had the same issue subsequent games

Head to Steam settings and enable "Allow downloads during gameplay"

**Comment by method0G** [comment]
Head to Steam settings and enable "Allow downloads during gameplay"

**Comment by basocjk** [comment]
> Head to Steam settings and enable "Allow downloads during gameplay"

this had the opposite effect on me.
enabling this setting causes the problem.
disabling solves the problem.

is it because i'm running steam as an admin?

**Comment by method0G** [comment]
> This is still occurring for me, both as Tinker and as Dazzle, potentially when previously queuing as core and now queuing as a support.

Running steam as administrator and allowing downloads during gameplay fixed it for me.

**Comment by mihaidraghici98** [comment]
Is there any way of debugging this? I have tried running as administrator, cleaning download caches, fresh reinstall of steam and dota2, nothing works.
This is a real deal breaker for the entire gameplay experience. Are you willing to look into this? It is quite old. Thanks @gdrewb-valve

**Comment by mihaidraghici98** [comment]
There are plenty of similar issues reporting guide cannot be loaded. Just check the issue search tool [here](https://github.com/ValveSoftware/Dota2-Gameplay/issues?q=is%3Aissue+is%3Aopen+guide)
Is there any way of debugging this, like checking what's the error thrown behind the scenes using the console?
cc: @BeeeQueue @jeffhill @danieljennings @wykrhm

**Comment by Timoses** [comment]
It works sporadically for me. I usually toggle between Downloads during gameplay checked and unchecked. Not sure if it even does anything.
While not playing a game (fresh entering Dota) it seems I can (almost?) always load Guides via "Heroes" -> <HERO> -> "Guide" menu.
But often not in-game of during Pick/Planning screen.

**Comment by method0G** [comment]
> It works sporadically for me. I usually toggle between Downloads during gameplay checked and unchecked. Not sure if it even does anything. While not playing a game (fresh entering Dota) it seems I can (almost?) always load Guides via "Heroes" -> -> "Guide" menu. But often not in-game of during Pick/Planning screen.

Yes, I thought it was fixed but turns out it's just very inconsistent. Sometimes it downloads on the menu and then carries over to multiplayer, but never the opposite.
